### PR TITLE
chore: update ComfyUI v0.19.3 → v0.25.0 with all vendored deps

### PR DESCRIPTION
**Pull request by jamesbrink**
## Summary

Updates ComfyUI from **v0.19.3** to **v0.25.0** (6 upstream releases: v0.20.1 through v0.25.0) and refreshes all vendored dependency pins.

### Core changes

- **ComfyUI** v0.19.3 → v0.25.0 (released 2026-06-16)
- **PyAV** wheel override 14.2.0 → 16.0.1 (upstream now requires `av>=16.0.0`)
- **New runtime dep**: `filelock` added to Python package list
- **Tightened constraints** in the pip-freeze shim: `transformers>=4.50.3`, `numpy>=1.25.0`, `safetensors>=0.4.2`, plus new pins for `aiohttp`, `yarl`, `SQLAlchemy`, `av`, `simpleeval`

### Vendored package bumps

| Package | Old | New |
|---|---|---|
| comfyui-frontend-package | 1.42.11 | 1.45.15 |
| comfyui-workflow-templates | 0.9.57 | 0.10.0 |
| comfyui-workflow-templates-core | 0.3.209 | 0.3.255 |
| comfyui-workflow-templates-media-api | 0.3.69 | 0.3.80 |
| comfyui-workflow-templates-media-video | 0.3.78 | 0.3.92 |
| comfyui-workflow-templates-media-image | 0.3.127 | 0.3.152 |
| comfyui-workflow-templates-media-other | 0.3.176 | 0.3.220 |
| comfyui-embedded-docs | 0.4.3 | 0.5.4 |
| comfyui-manager | 4.1 | 4.2.2 |
| comfy-kitchen | 0.2.8 | 0.2.10 |
| comfy-aimdo | 0.2.12 | 0.4.10 |

### Housekeeping

- **Removed** obsolete `nix/patches/comfyui-ltxvideo-rotary-emb.patch` — the upstream LTXVideo rotary embedding issue was fixed in a prior ComfyUI release
- **Fixed** the `awk` snippet printed by `nix run .#update` so it no longer fails under `set -u` (unescaped `$2` in the `writeShellApplication`)
- **Regenerated** `nix/template-inputs.nix` (589 workflow template input files)
- **Updated** `CHANGELOG.md` with full version entry

### Upstream highlights (v0.20.1 → v0.25.0)

- New and updated 3D, video, audio, asset-management, and API node features
- Cursor-based asset pagination, image dimension metadata, asset IDs in execution messages, `/system_stats` deploy environment reporting
- New CLI flags: `--debug-hang`, `--high-ram`, dynamic `--reserve-vram`, `--vram-headroom`
- Model support: SCAIL-2, Bernini-R, Depth Anything 3, LTX/Wan/Qwen/Ideogram improvements, 10-bit video
- Fixes for video decode edge cases, pinned-memory pressure, quantized weights, dtype handling, manager enablement behavior

## Test plan

- [ ] `nix build .#packages.x86_64-linux.default --no-link` succeeds
- [ ] `nix flake check` passes (ruff, pyright, nixfmt, shellcheck, full build)
- [ ] `nix run .#update` prints the correct latest version without `set -u` errors
- [ ] Runtime startup on a disposable data dir with `--cpu` completes without errors
- [ ] Queue `EmptyImage → SaveImage` and verify a generated 64×64 PNG is written
- [ ] Model downloader smoke test: download a test file, confirm progress reaches `completed`
- [ ] Docker image builds: `nix run .#buildDockerLinux` (CPU), `nix run .#buildDockerLinuxCuda` (CUDA)

### CODE REVIEW

File: nix/python-overrides.nix [hunk 1]
```diff
@@ -813,27 +813,27 @@ lib.optionalAttrs useCuda {
...
-            hash = "sha256-amqunheq5PKpczWCXApwG3Y7cqr4lCjypwu9yDtkrSM=";
+            url = "https://files.pythonhosted.org/packages/44/78/12a11d7a44fdd8b26a65e2efa1d8a5826733c8887a989a78306ec4785956/av-16.0.1-cp312-cp312-macosx_11_0_x86_64.whl";
+            hash = "sha256-5BqP74XfsscXNJ+f90+S+VYBIqnxqUscbJqKnJRiunE=";
           }
         else if pkgs.stdenv.isDarwin && pkgs.stdenv.hostPlatform.isAarch64 then
           pkgs.fetchurl {
-            url = "https://files.pythonhosted.org/packages/5b/88/b56f5e5fa2486ee51413b043e08c7f5ed119c1e10b72725593da30adc28f/av-14.2.0-cp312-cp312-macosx_12_0_arm64.whl";
-            hash = "sha256-o9o+lRFIKR1w9ss/s3v4FYCwGZLpFe8QMBCOQHb2LTg=";
+            url = "https://files.pythonhosted.org/packages/27/19/3a4d3882852a0ee136121979ce46f6d2867b974eb217a2c9a070939f55ad/av-16.0.1-cp312-cp312-macosx_14_0_arm64.whl";
+            hash = "sha256-Y1KmSyXJ+YXU8nnCkC25qSQk5vLJchYeZxGWFvB5bLk=";
```
[the remark this answers is not in the document]
Comment (jamesbrink) [reply nix/python-overrides.nix:836]: Won't fix — PyAV 16.0.1 only publishes a single aarch64-darwin wheel (`macosx_14_0_arm64`); there is no lower deployment-target alternative available on PyPI. Since upstream ComfyUI v0.25.0 requires `av>=16.0.0`, this is a hard constraint from the upstream ecosystem, not a choice we can change by picking a different wheel.

macOS 14 (Sonoma) was released October 2023 and is the oldest macOS version still receiving security updates from Apple. The practical impact is minimal — users on older macOS would already be running unsupported OS versions. This effectively becomes a documented minimum for Apple Silicon builds.
